src/transactions/feed/detailContent/TransferSentContent.tsx

```diff
@@ -21,16 +21,19 @@ import networkConfig from 'src/web3/networkConfig'
 
 // Note that this is tested from TransactionDetailsScreen.test.tsx
 function TransferSentContent({ transfer }: { transfer: TokenTransfer }) {
-  const { amount, metadata, address } = transfer
```

Comment:
i was confused whether to use `transfer.amount` or `amount`, this component is inconsistent. it was less changes to remove the destructure than remove usages of `transfer.`